src/services/activityReports.js

```diff
@@ -364,7 +346,7 @@ export async function populateRecipientInfo(activityRecipients, grantPrograms) {
         recipient,
         'name',
         {
-          value: nameValue,
+          value: recipient.name,
```

Comment:
hmm why do we need to set the recipient's name to what it already is?

Reply:
Will remove.